api/src/queries/project-queries.ts

```diff
@@ -589,3 +651,88 @@ export const postProjectIUCNSQL = (iucn_id: number, project_id: number): SQLStat
 
   return sqlStatement;
 };
+
+/**
+ * SQL query to insert a project activity row.
+ *
+ * @param activityId
+ * @param projectId
+ * @returns {SQLStatement} sql query object
+ */
+export const postProjectActivitySQL = (activityId: number, projectId: number): SQLStatement | null => {
+  defaultLog.debug({
+    label: 'postProjectActivity',
+    message: 'params',
+    activityId,
+    projectId
+  });
+
+  if (!activityId || !projectId) {
+    return null;
+  }
+
+  const sqlStatement: SQLStatement = SQL`
+      INSERT INTO project_activity (
+        a_id,
+        p_id
+      ) VALUES (
+        ${activityId},
+        ${projectId}
+      )
+      RETURNING
+        id;
+    `;
+
+  defaultLog.debug({
+    label: 'postProjectActivity',
+    message: 'sql',
+    'sqlStatement.text': sqlStatement.text,
+    'sqlStatement.values': sqlStatement.values
+  });
+
+  return sqlStatement;
+};
+
+/**
+ * SQL query to insert a project activity row.
```

Comment:
this comment seems to be a copy pasta error 😄

Reply:
Updated doc